sympy/geometry/tests/test_polygon.py

```diff
@@ -405,3 +405,12 @@ def test_intersection():
     assert poly2.intersection(Triangle(Point(0, 1), Point(1, 0), Point(-1, 1))) == [Point(-5/7, 6/7),
                                                                                     Segment(Point2D(0, 1), Point(1, 0))]
     assert poly1.intersection(RegularPolygon((-12, -15), 3, 3)) == []
+
+def test_parameter_value():
+    sq = Polygon((0,0),(0,1),(1,1),(1,0))
```

Comment:
More missing spaces here. If you don't have PEP8 checker installed, I suggest http://pep8online.com/  which is sufficient for checking short blocks of code.

Reply:
My suggestion of less trivial tests applies here as well. I took a random triangle and a random point:
```
Q = Polygon((3, 4), (5, 5), (0, 0))
Q.parameter_value((7, 7))
```
and got unhandled RuntimeError

Reply:
@normalhuman ```solve``` is not able to handle complex equations. ```solveset``` is also giving error in some cases. I think we should raise an exception if ```solve()``` is not able to solve equation and create a new issue regarding this.